packages/explorer/src/bin/explorer.ts

```diff
@@ -97,6 +101,30 @@ async function startExplorer() {
   }
 }
 
+async function startStoreIndexer() {
+  if (chainId !== anvil.id) {
+    console.log("Skipping SQLite indexer for non-anvil chain ID", chainId);
+    return;
+  }
+
+  if (existsSync(indexerDatabasePath)) {
+    execSync(`shx rm -rf ${indexerDatabasePath}`);
+  }
+
+  console.log("Running SQLite indexer for anvil...");
+  indexerProcess = spawn("sh", ["node_modules/.bin/sqlite-indexer"], {
+    cwd: packageRoot,
+    stdio: "inherit",
+    env: {
```

Comment:
should prob pass in a `STORE_ADDRESS` to ensure we're only indexing the targeted world

Reply:
oh that's neat, added `STORE_ADDRESS` env var as world address

Reply:
since we're parsing the worlds.json, we can also pass in `START_BLOCK` to speed things up (esp helpful when we start supporting remote chains)

Reply:
could you clarify what you mean, or point to how to obtain the value for `START_BLOCK`, or should it be passed explicitly? Looked into `worlds.json` but it only contains world address.

Reply:
oh right, I forgot we don't write the start block for local chains because it creates a lot of noise with git seeing the start block changing after each new deploy (whereas the world address is more stable because its deterministically deployed)

Reply:
cool, will do!